src/AbiBinProvider.js

```diff
@@ -2,41 +2,18 @@
 
 const mosaicContracts = require('@openstfoundation/mosaic-contracts');
 
-//__NOT_FOR_WEB__BEGIN__
-const fs = require('fs'),
-  path = require('path');
-//__NOT_FOR_WEB__END__
-
-let DEFAULT_ABI_FOLDER_PATH, DEFAULT_BIN_FOLDER_PATH;
-//__NOT_FOR_WEB__BEGIN__
-DEFAULT_ABI_FOLDER_PATH = path.resolve(__dirname, '../contracts/abi/');
-DEFAULT_BIN_FOLDER_PATH = path.resolve(__dirname, '../contracts/bin/');
-//__NOT_FOR_WEB__END__
-
+const { loadContracts } = require('./AbiBinProvider-node.js');
 const Linker = require('../src/utils/linker');
 
 class AbiBinProvider {
   constructor(abiFolderPath, binFolderPath) {
     const oThis = this;
-    abiFolderPath = abiFolderPath || DEFAULT_ABI_FOLDER_PATH;
-    binFolderPath = binFolderPath || DEFAULT_BIN_FOLDER_PATH;
-    //__NOT_FOR_WEB__BEGIN__
-    if (!path.isAbsolute(abiFolderPath)) {
-      let err = new Error(
-        '"abiFolderPath" is not Absolute. Please provide absolute path.',
-      );
-      throw err;
-    }
-    if (!path.isAbsolute(binFolderPath)) {
-      let err = new Error(
-        '"binFolderPath" is not Absolute. Please provide absolute path.',
-      );
-      throw err;
+
+    // only does something on nodejs
```

Comment:
```suggestion
    // loadContracts() returns an empty object in the browser.
```

Reply:
It doesn't return an empty object, it is a noop.

Reply:
Ha! Still learning to read JS 😅 
The comment should still be an English sentence with a period.